mock/veda.config.js

```diff
@@ -103,10 +118,24 @@ module.exports = {
     }
   },
   banner: {
-    text: 'Read the new data insight on using EMIT and AVIRIS-3 for monitoring large methane emission events.',
-    url: 'stories/emit-and-aviris-3',
-    expires: '2024-08-03T12:00:00-04:00',
-    type: 'info'
+    headerText: 'An official website of the United States government',
+    headerActionText: "Here's how you know",
+    ariaLabel: 'Banner for official government website',
+    flagImgSrc: '/img/us_flag_small.png',
+    flagImgAlt: 'US flag',
+    leftGuidance: defaultGuidance.left,
+    rightGuidance: defaultGuidance.right,
+    className: '',
+    defaultIsOpen: false,
+    contentId: 'gov-banner-content'
+  },
+  siteAlert: {
+    content:
+      '<p class="usa-alert__text"><a target="_blank" rel="noreferrer" href="stories/emit-and-aviris-3">Read the new data insight</a> on using EMIT and AVIRIS-3 for monitoring large methane emission events.</p>',
+    expires: '2026-08-03T12:00:00-04:00',
```

Comment:
This is just mock but clicking on it leads to Page not found since we dont have this story in veda-ui. Just an FYI if you weren't already aware since I dont know if we want